pom.xml

```diff
@@ -27,12 +58,6 @@
 			<version>${junit-jupiter-version}</version>
 			<scope>test</scope>
 		</dependency>
-		<dependency>
```

Comment:
Removed the junit platform jar that is junit 4. I've migrated all tests to junit 5 too.